python-package/README.rst

```diff
@@ -54,77 +61,98 @@ To install all dependencies needed to use ``pandas`` in LightGBM, append ``[pand
 
     pip install 'lightgbm[pandas]'
 
+|
+
+Use LightGBM with Matplotlib and Graphviz
+*****************************************
+
+To install all dependencies needed to use ``lightgbm.plotting``, append ``[plotting]``.
+
+.. code:: sh
+
+    pip install 'lightgbm[plotting]'
+
+|
+
 Use LightGBM with scikit-learn
 ******************************
 
-To install all dependencies needed to use ``scikit-learn`` in LightGBM, append ``[scikit-learn]``.
+To install all dependencies needed to use ``lightgbm.sklearn``, append ``[scikit-learn]``.
 
 .. code:: sh
 
     pip install 'lightgbm[scikit-learn]'
 
+|
+
 Build from Sources
 ******************
 
 .. code:: sh
 
-    pip install --no-binary lightgbm lightgbm
-
-Also, in some rare cases you may need to install OpenMP runtime library separately (use your package manager and search for ``lib[g|i]omp`` for doing this).
+    pip install lightgbm --no-binary lightgbm
 
 For **macOS** users, you can perform installation either with **Apple Clang** or **gcc**.
 
 - In case you prefer **Apple Clang**, you should install **OpenMP** (details for installation can be found in `Installation Guide <https://github.com/microsoft/LightGBM/blob/master/docs/Installation-Guide.rst#apple-clang>`__) first.
 
-- In case you prefer **gcc**, you need to install it (details for installation can be found in `Installation Guide <https://github.com/microsoft/LightGBM/blob/master/docs/Installation-Guide.rst#gcc>`__) and specify compilers by running ``export CXX=g++-7 CC=gcc-7`` (replace "7" with version of **gcc** installed on your machine) first.
+- In case you prefer **gcc**, you need to install it (details for installation can be found in `Installation Guide <https://github.com/microsoft/LightGBM/blob/master/docs/Installation-Guide.rst#gcc-1>`__) and specify compilers by running ``export CXX=g++-7 CC=gcc-7`` (replace "7" with version of **gcc** installed on your machine) first.
 
 For **Windows** users, **Visual Studio** (or `VS Build Tools <https://visualstudio.microsoft.com/downloads/>`_) is needed.
 
+|
+
 Build Threadless Version
 ~~~~~~~~~~~~~~~~~~~~~~~~
 
 .. code:: sh
 
-    pip install lightgbm --config-settings=cmake.define.USE_OPENMP=OFF
+    pip install lightgbm --no-binary lightgbm --config-settings=cmake.define.USE_OPENMP=OFF
```

Comment:
Without `--no-binary` flag `pip` doesn't try to build from sources but simply installs precompiled wheel file regardless specified `--config-settings` flags.